src/lib/components/Subscribe/index.js

```diff
@@ -37,6 +37,28 @@ class Subscribe extends BaseElement {
     super.connectedCallback();
     /** @type {HTMLFormElement} */
     this.form = this.querySelector('form');
+    const intersectionObserver = new IntersectionObserver(([entry]) => {
+      if (!entry.isIntersecting) {
+        return;
+      }
+
+      intersectionObserver.disconnect();
+      window.recaptchaLoadCallback = () => {
+        /** @type {HTMLDivElement} */
+        const recaptchaContainerEl = this.querySelector('.g-recaptcha');
+        recaptchaContainerEl.children[0].remove();
```

Comment:
I wonder how this would work in Percy 🤔 we had a problem with double-initing components on d.c.c. - maybe it makes sense to guard all this by checking for the `unresolved` class the BaseElement adds?